src/main/resources/add-ons-to-index.json

```diff
@@ -2103,6 +2208,9 @@
       "uid": "org.openmrs.module.patientflags",
       "type": "OMOD",
       "name": "Patient Flags",
+	  "tags": [
+        "patients"
```

Comment:
not a useful tag. (Almost everything in OpenMRS is really about patients, so this doesn't give useful info)